src/main/shared/util/util-model.ts

```diff
@@ -0,0 +1,3 @@
+export interface IObject {
```

Comment:
@dolanmiu I made this to avoid using 'object' that eslint forbids but doesn't really like it. Any suggestion? Not sure about best practices.

Reply:
what are we trying to achieve? yes eslint is correct because object is not typed :D 

I think generics is maybe the answer `<T>`, but i dont know the context

Reply:
Several places I just wanted to make an object with key value pairs without bothering the value type